app/routes/api.js

```diff
@@ -3,37 +3,69 @@
 var express = require('express');
 var router = express.Router();
 
+var Account = require('../models/account');
 var Log = require('../models/log');
 
-// logs ======================================================================
-router.post('/logs', function(req, res) {
-    var userId = req.body.userId;
+module.exports = function(passport) {
+    // logs ======================================================================
+    router.post('/logs/', passport.authenticate('bearer', { session: false }), function(req, res) {
+        // force application/json content-type
+        if(req.headers["content-type"] !== 'application/json') {
+            return res.status(400).send('Error: content-type must be application/json');
```

Comment:
will this send plaintext of json? if we're defining a json only endpoint, we should return json errors i think